README.md

```diff
@@ -77,6 +77,18 @@ lh.move_plate(lh.deck.get_resource("plate"), pr)
 data = await pr.read_luminescence()
 ```
 
+### Centrifuges
+
+PyLabRobot also supports centrifugation (spin spin spin), currently limited to the Agilent Centrifuge. This functionality allows users to automate centrifuging processes on any modern operating system. Here is a quick example of centrifugation at 800 g-force for 60 seconds:
```

Comment:
i know we make this mistake elsewhere to, but the readme would get very long if we always start with 'plr also supports x'. the paragraph above can be shortened to: "Spinning at 800 g for 60 seconds using an Agilent VSpin"

(i shall take care of other such instances)